websites/views_test.py

```diff
@@ -680,6 +680,11 @@ def test_website_starters_site_configs_exception(mocker, drf_client):
         ["", "page", "", 5],
         ["", "", "text2", 1],
         ["", "", "", 6],
+        ["", "", "myfile3", 1],
+        ["", "", "myfile", 5],
+        ["", "", ".png", 5],
+        ["", "", ".p", 5],
+        ["", "", ".pdf", 0],
```

Comment:
Add some tests to make sure a search matching only the path part of a resource file won't be returned (see comment below):

```python
["", "", "test", 0],
["", "", "courses", 0],
```